src/internal/account.ts

```diff
@@ -137,12 +141,45 @@ export async function getTransactions(args: {
   options?: PaginationArgs;
 }): Promise<TransactionResponse[]> {
   const { aptosConfig, accountAddress, options } = args;
-  return paginateWithCursor<{}, TransactionResponse[]>({
+  // TODO: Ideally indexer should provide one API that returns all transactions details in one go
+  // But now we have to query all transaction versions first and then query each transaction by version
+  const versions = await getAccountAllTransactionVersions({ aptosConfig, accountAddress, options });
```

Comment:
can we keep this query as is, and add another query like `getAllUserTransactions` that does this multiple queries? and mention it on the function comment